unison-src/transcripts-manual/benchmarks/tmap.u

```diff
@@ -13,12 +13,10 @@ main = do
   _a7 = timeit "TVar Map concurrent (4)" '(gos_TVar_Map 4 (atomically '(TVar.new Map.empty)) N)
   _a8 = timeit "TVar Map concurrent (8)" '(gos_TVar_Map 8 (atomically '(TVar.new Map.empty)) N)
   _a9 = timeit "TVar Map concurrent (16)" '(gos_TVar_Map 16 (atomically '(TVar.new Map.empty)) N)
-  _a10 = timeit "TVar Map concurrent (32)" '(gos_TVar_Map 32 (atomically '(TVar.new Map.empty)) N)
```

Comment:
this benchmark was taking too long and we weren't seeing any nonlinear slowdown with more concurrency.